commit 3bdc30e8bf3e10e6b6b283d216806d8e5d333c1a
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-03-05T18:47:21+00:00 (7 months ago)
    
    TOOLS-2193 TOP Makefile.defs assignment complicates overriding
    Reviewed by: Tim Foster <tim.foster@joyent.com>
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Tim Foster <tim.foster@joyent.com>

diff --git a/tools/mk/Makefile.defs b/tools/mk/Makefile.defs
index fa3569e..8bd6ac9 100644
--- a/tools/mk/Makefile.defs
+++ b/tools/mk/Makefile.defs
@@ -53,7 +53,7 @@
 # destinations, but sometimes to modify the behavior of the build.
 #
 
-TOP := $(shell pwd)
+TOP ?= $(shell pwd)
 
 #
 # Mountain Gorilla-spec'd versioning.
